### ISSUE TITLE
.Net Bug: Failed to build Concepts project

### ISSUE THREAD

**Issue by Krzysztof318** [microsoft/semantic-kernel#7017]
**Describe the bug**
Building Concepts project fails with errors:
> E:\Projects\Forks\semantic-kernel\dotnet\samples\Concepts\obj\Debug\net8.0\.NETCoreApp,Version=v8.0.AssemblyAttributes.cs(4,12): error CS0579: Duplicate 'global::System.Runtime.Versioning.TargetFrameworkAttribute' attribute  
[E:\Projects\Forks\semantic-kernel\dotnet\samples\Concepts\Concepts.csproj]
E:\Projects\Forks\semantic-kernel\dotnet\samples\Concepts\obj\Debug\net8.0\Concepts.AssemblyInfo.cs(13,12): error CS0579: Duplicate 'System.CLSCompliantAttribute' attribute [E:\Projects\Forks\semantic-kernel\dotnet\samples\C 
oncepts\Concepts.csproj]
E:\Projects\Forks\semantic-kernel\dotnet\samples\Concepts\obj\Debug\net8.0\Concepts.AssemblyInfo.cs(14,12): error CS0579: Duplicate 'Microsoft.Extensions.Configuration.UserSecrets.UserSecretsIdAttribute' attribute [E:\Projec 
ts\Forks\semantic-kernel\dotnet\samples\Concepts\Concepts.csproj]
E:\Projects\Forks\semantic-kernel\dotnet\samples\Concepts\obj\Debug\net8.0\Concepts.AssemblyInfo.cs(15,12): error CS0579: Duplicate 'System.Reflection.AssemblyCompanyAttribute' attribute [E:\Projects\Forks\semantic-kernel\do 
tnet\samples\Concepts\Concepts.csproj]
E:\Projects\Forks\semantic-kernel\dotnet\samples\Concepts\obj\Debug\net8.0\Concepts.AssemblyInfo.cs(16,12): error CS0579: Duplicate 'System.Reflection.AssemblyConfigurationAttribute' attribute [E:\Projects\Forks\semantic-ker 
nel\dotnet\samples\Concepts\Concepts.csproj]
E:\Projects\Forks\semantic-kernel\dotnet\samples\Concepts\obj\Debug\net8.0\Concepts.AssemblyInfo.cs(17,12): error CS0579: Duplicate 'System.Reflection.AssemblyFileVersionAttribute' attribute [E:\Projects\Forks\semantic-kerne 
l\dotnet\samples\Concepts\Concepts.csproj]
E:\Projects\Forks\semantic-kernel\dotnet\samples\Concepts\obj\Debug\net8.0\Concepts.AssemblyInfo.cs(18,12): error CS0579: Duplicate 'System.Reflection.AssemblyInformationalVersionAttribute' attribute [E:\Projects\Forks\seman 
tic-kernel\dotnet\samples\Concepts\Concepts.csproj]
E:\Projects\Forks\semantic-kernel\dotnet\samples\Concepts\obj\Debug\net8.0\Concepts.AssemblyInfo.cs(19,12): error CS0579: Duplicate 'System.Reflection.AssemblyProductAttribute' attribute [E:\Projects\Forks\semantic-kernel\do 
tnet\samples\Concepts\Concepts.csproj]
E:\Projects\Forks\semantic-kernel\dotnet\samples\Concepts\obj\Debug\net8.0\Concepts.AssemblyInfo.cs(20,12): error CS0579: Duplicate 'System.Reflection.AssemblyTitleAttribute' attribute [E:\Projects\Forks\semantic-kernel\dotn 
et\samples\Concepts\Concepts.csproj]
E:\Projects\Forks\semantic-kernel\dotnet\samples\Concepts\obj\Debug\net8.0\Concepts.AssemblyInfo.cs(21,12): error CS0579: Duplicate 'System.Reflection.AssemblyVersionAttribute' attribute [E:\Projects\Forks\semantic-kernel\do 
tnet\samples\Concepts\Concepts.csproj]

**To Reproduce**
1. Pull latest main.
2. Try build samples/Concepts project.

**Platform**
 - OS: [Windows]
 - Language: [C#]
 - Source: [latest main]

**Additional context**
Simple way to fix this is adding these lines to Concepts.csproj
```csharp
<GenerateAssemblyInfo>false</GenerateAssemblyInfo>
<GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
```
But it should build normally even without this, other projects don't have this issue.

**Comment by RogerBarreto** [comment]
TLDR

Delete KernelSyntaxExamples and KernelAgentSyntax empty projects within your existing old structure.

This only happened when we migrated the KernelExampleSyntax projects to the new one.

This raises because git client don't delete the KernelSyntaxExamples as well as don't delete `AgentExamples` empty folders as they have residue `bin`, `obj` folders within.
